website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,167 @@
+
+
+## Variants
+
+### Single
+Selection is limited to a single list item.
+
+![Single selection example with list open](/assets/components/form/super-select/super-select-single-variant-text.png =504x*)
+
+### Multiple
+Multiple selections can be made within the list.
+
+![Multiple selection example with list open](/assets/components/form/super-select/super-select-multiple-variant.png =504x*)
+
+## Trigger default states
+
+**Empty**
+The default option, where the trigger is empty, showing nothing selected.
+
+![Showing an empty trigger, indicating to the user there is no option selected yet](/assets/components/form/super-select/super-select-trigger-empty.png =399x*)
+
+**Placeholder text**
+Placeholder text was added to the trigger, which helps the user understand additional context, but is not reliant on this information to carry out a task.
+
+![Showing placeholder text inside of the trigger, still indicating no selection is made, but with some additional guidance](/assets/components/form/super-select/super-select-trigger-placeholder-text.png =399x*)
+
+**Populated**
+The trigger shows information populated, and has an option pre-selected before user interaction.
+
+![Showing preselected options in both a single and multiple variant. This helps show something has been preselected previously and the user can update their options](/assets/components/form/super-select/super-select-trigger-filled.png =399x*)
+
+## Before and after options
+The before and after options are areas that sit above and below the list respectively, independent from the list’s overflow. They allow for either predefined components (like search) or custom content to occupy these spaces.
+
+### BeforeOptions
+BeforeOptions allows for one of two options to be displayed at a given time: search or custom content. These options are the same between both single and multiple variants.
+
+#### Search
+If enabled, allows the user to search by filtering the results based on the query.
+
+![Search enabled before the list in both single and multiple variants](/assets/components/form/super-select/super-select-before-options-search.png =836x*)
+
+##### Search to add behavior
+If enabled, searched terms can be added to the list. This feature is only applicable to the multiple variant.
+![Showing the search to add behavior in the multiple variant of SuperSelect](/assets/components/form/super-select/super-select-multi-flow-search-to-add.png =836x*)
+
+#### CustomContent
+If enabled, yielded content will occupy this space.
+
+![Custom content enabled before the list in both single and multiple variants](/assets/components/form/super-select/super-select-before-options-custom-content.png =836x*)
+
+!!! Dont
+
+We recommend against placing heavy UI elements in the custom content as it can draw attention away from the list options and will only be visible when the menu is open. Consider placing them in relation to the trigger.
```

Comment:
Updated this to say:

"We recommend against placing heavy UI elements in custom content as it can draw attention away from the list options and will only be visible when the menu is open. Consider placing them in outside the list relative to the trigger for better visibility."